cf/irsa-api-role.yaml

```diff
@@ -109,6 +109,7 @@ Resources:
                   - "states:CreateStateMachine"
                   - "states:UpdateStateMachine"
                   - "states:StartExecution"
+                  - "states:states:TagResource"
```

Comment:
do you really need the double `states` in here?

Reply:
OMG :-(

Reply:
done. Thanks!